SekoiaIO/endpoint/_meta/fields.yml

```diff
@@ -6862,4 +6862,239 @@ sekoiaio.target_process.name:
   type: keyword
   observable:
     type: file
-    property: name
\ No newline at end of file
+    property: name
+
+action.properties.a0:
+  name: action.properties.a0
+  type: keyword
+  description: "argument 0 of syscall"
+
+action.properties.a1:
+  name: action.properties.a1
+  type: keyword
+  description: "argument 1 of syscall"
+
+action.properties.a2:
+  name: action.properties.a2
+  type: keyword
+  description: "argument 2 of syscall"
+
+action.properties.a3:
+  name: action.properties.a3
+  type: keyword
+  description: "argument 3 of syscall"
+
+action.properties.acct:
+  name: action.properties.acct
+  type: keyword
+  description: "account related to the event"
+
+action.properties.addr:
+  name: action.properties.addr
+  type: keyword
+  description: ""
```

Comment:
As this is a work for one of our product, it would be nice to complete every descriptions

Reply:
Some of theses keys are application specific, I'm not sure that it cannot be used by something else. As an exemple, in the context of an `ssh` command, the `size` attribute means the size of the key used. But I think the term is generic enough to be used by other applications.